### ISSUE TITLE
Formatting on windows Powershell very off

### ISSUE THREAD

**Issue by EinAeffchen** [sherlock-project/sherlock#5]
[37;1m[[92;1m+[37;1m][92;1m Unsplash:[0m https://unsplash.com/@User
[1;92m[[0m[1;77m*[0m[1;92m] Saved: [37;1mUser.txt[0m

That's how all the lines look like in windows powershell. I guess the print is designed for a linux platform?

**Comment by sdushantha** [comment]
I have heard that [Hue](https://github.com/s0md3v/hue) is very good. 
Which one should we use? Hue or Colorama?

**Comment by Calinou** [comment]
Hue's README says:

> **Note:** Windows versions below windows 10 do not support ANSI escape sequences so the colors will not be printed in command prompt.

Colorama can also be used to wrap other color libraries on Windows (it will do nothing on other platforms), so you can use *both*, actually.

**Comment by hoerenraph** [comment]
hue works perfectly
run `pip install huepy`
add `from huepy import *` to the importing part of the sherlock.py file

**Comment by jcs090218** [comment]
I personally think `Colorama`'s style are better than `Hue`'s style. Below are the comparison of two libraries.

#### Hue ####
```
from hue import *
print red('This string is red')
```

#### Colorama ####
```
from colorama import Fore
print Fore.RED + 'This string is red'
```

The point here, if use `Hue`, the code might end up like `print(red('This string is red'))`. On the other hand, if we use `Colorama` the code likeliest be like `print(Fore.RED + 'This string is red')`. Unless the code are not going to change to something more readable, I reckon this would not matter.

**Comment by nareddyt** [comment]
Closed via #71

**Comment by sdushantha** [comment]
Closing this issue as it has been fixed. There is also an option **not** not have colors which was introduced in #463
